src/wpt/harness.ts

```diff
@@ -67,13 +119,7 @@ globalThis.subsetTestByKey = (_key, testType, testCallback, testMessage) => {
   return testType(testCallback, testMessage);
 };
 
-globalThis.promise_test = async (func, name, properties) => {
-  if (typeof func !== 'function') {
-    properties = name;
-    name = func;
-    func = null;
-  }
```

Comment:
I'm not sure what's going on here. If this branch is taken, we'd eventually get `null.call(this)` on line 128, which I don't think is good?

Reply:
This code was taken from the actual web-platform-tests WPT harness. There might be a bug somewhere, or probably a porting error.

Reply:
I can confirm it's in the original code. I'll leave it out for now until we find an example of a non-function being passed, and then we can upgrade our harness to deal with it.